src/driver/amdxdna/aie2_pm.c

```diff
@@ -68,7 +68,7 @@ int aie2_pm_set_mode(struct amdxdna_dev_hdl *ndev, enum amdxdna_power_mode_type
 
 	switch (target) {
 	case POWER_MODE_TURBO:
-		if (ndev->ctx_num) {
```

Comment:
it might make more sense to use ctx_cnt here.

Reply:
fixed